udata/frontend/helpers.py

```diff
@@ -35,6 +36,12 @@ def now():
     return datetime.now()
 
 
+def cdn_for(endpoint, **kwargs):
```

Comment:
Not a fan of the name. `cdn_or_flask_url_for` ? Not great either :-/ but more descriptive.

Reply:
This is only meant to be used for CDN assets, not as a replacement for all flask `url_for`.
I kept the name but added a docstring to be more explicit.